src/components/ComparisonContainer/index.tsx

```diff
@@ -0,0 +1,504 @@
+                <h2>Remove User</h2>
+                <p>
+                  Are you sure you want to remove the following user&apos;s
+                  schedules from your view?
+                </p>
+                <p>
+                  User: <b>{deleteConfirm?.name}</b>
+                </p>
+                <p>
+                  You will not be able to see any of their schedules unless the
+                  owner sends another invite for each one.
+                </p>
+              </div>
+            )}
+            {deleteConfirm?.type === 'Schedule' && (
+              <div style={{ textAlign: 'center' }}>
+                <h2>Remove Schedule</h2>
+                <p>
+                  Are you sure you want to remove the following schedule from
+                  your view?
+                </p>
+                <p>
+                  Schedule: <b>{deleteConfirm?.name}</b> <br />
+                  Owner: <b>{deleteConfirm?.owner}</b>
+                </p>
+                <p>
+                  You will not be able to see it unless the owner sends another
+                  invite.
+                </p>
+              </div>
+            )}
+          </Modal>
+        </div>
+        <div
+          className={classes('comparison-overlay', 'left', compare && 'open')}
+        />
+        <div
+          className={classes('comparison-overlay', 'right', !compare && 'open')}
+        />
+      </div>
+    </div>
+  );
+}
+
+type ScheduleRowProps = {
+  id: string;
+  type: string;
+  owner?: string;
+  hasCheck?: boolean;
+  onClick?: () => void;
+  checkboxColor?: string;
+  name: string;
+  handleEditSchedule: () => void;
+  handleRemoveSchedule: () => void;
+  hasDelete?: boolean;
+  hasTooltip?: boolean;
+  editOnChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  editOnKeyDown: (e: React.KeyboardEvent<HTMLInputElement>) => void;
+  editInfo: EditInfo;
+  editValue: string;
+};
+
+function ScheduleRow({
+  id,
+  type,
+  owner,
+  hasCheck = true,
+  onClick,
+  checkboxColor,
+  name,
+  handleEditSchedule,
+  handleRemoveSchedule,
+  hasDelete = true,
+  hasTooltip = false,
+  editOnChange,
+  editOnKeyDown,
+  editInfo,
+  editValue,
+}: ScheduleRowProps): React.ReactElement {
+  const tooltipId = useId();
+  const [hover, setHover] = useState(false);
+
+  const edit =
+    editInfo != null &&
+    editInfo.type === type &&
+    editInfo.id === id &&
+    editInfo.owner === owner;
+
+  return (
+    <div className="checkbox-container">
+      {hasCheck && (
+        <div
+          className={classes('checkbox', type === 'Schedule' && 'indented')}
+          onClick={onClick}
+          style={{ backgroundColor: checkboxColor }}
+        />
+      )}
+      {edit && (
+        <AutoFocusInput
+          className={classes('edit-input', hasCheck && 'check')}
+          value={editValue}
+          onChange={editOnChange}
+          placeholder={name}
+          onKeyDown={editOnKeyDown}
+        />
```

Comment:
The text input field does not disappear unless we click enter. Could you make the text input field disappear when the user clicks away (i.e., input field is no longer in focus)?
<img width="250" alt="image" src="https://user-images.githubusercontent.com/38119460/224235882-3443e84e-9291-4e53-bb1d-53a62be3e60c.png">